projects/.metadata/.plugins/org.eclipse.core.resources/.history/25/00a1e777f21d00181353d905c66b77c4

```diff
@@ -0,0 +1,384 @@
+*    notice, this list of conditions and the following disclaimer.
+*  - Redistributions in binary form must reproduce the above copyright
+*    notice, this list of conditions and the following disclaimer in
+*    the documentation and/or other materials provided with the
+*    distribution.
+*  - Neither the name of Analog Devices, Inc. nor the names of its
+*    contributors may be used to endorse or promote products derived
+*    from this software without specific prior written permission.
+*  - The use of this software may or may not infringe the patent rights
+*    of one or more patent holders.  This license does not release you
+*    from the requirement that you obtain separate licenses from these
+*    patent holders to use this software.
+*  - Use of the software either in source or binary form, must be run
+*    on or directly connected to an Analog Devices Inc. component.
+*
+* THIS SOFTWARE IS PROVIDED BY ANALOG DEVICES "AS IS" AND ANY EXPRESS OR IMPLIED
+* WARRANTIES, INCLUDING, BUT NOT LIMITED TO, NON-INFRINGEMENT, MERCHANTABILITY
+* AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+* IN NO EVENT SHALL ANALOG DEVICES BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+* SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+* INTELLECTUAL PROPERTY RIGHTS, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+* LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+* ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+* (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+* SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*
+*******************************************************************************/
+
+#include <sys/platform.h>
+#include "ADuCM3029_demo_cn0398.h"
+#include <common.h>
+#include <common/adi_timestamp.h>
+#include "Timer.h"
+
+using namespace adi_sensor_swpack;
+
+/* Static data */
+static ADI_DATA_PACKET    			gSensorData;
+static ADI_STRING_DATA				gCommData = {
+													.nStringSize = 13u};
+static ADI_WIFI_TCP_CONNECT_CONFIG  gTCPConnect = {
+													.nLinkID = ADI_WIFI_CONNECTION_ID,
+													.pType = (uint8_t *) "TCP",
+													.pIP   = aMQTTBrokerIp,
+													.pPort =  aMQTTBrokerPort,
+													.nTCPKeepAlive = ADI_WIFI_MQTT_PUBLISHER_KEEPALIVE};
+
+static ADI_WIFI_MQTT_CONNECT_CONFIG gMQTTConnect = {
+													.nLinkID = ADI_WIFI_CONNECTION_ID,
+													.nVersion = ADI_WIFI_MQTT_PUBLISHER_VERSION,
+													.pName = aMQTTPublisherName,
+													.nMQTTKeepAlive = ADI_WIFI_MQTT_PUBLISHER_KEEPALIVE };
+
+static ADI_WIFI_PUBLISH_CONFIG 		gPublishConfig = {
+		 	 	 	 	 	 	 	 	 	 	 	 .pMQTTData = (uint8_t *) &gSensorData,
+		 	 	 	 	 	 	 	 	 	 	 	 .nMQTTDataSize = sizeof(ADI_DATA_PACKET),
+													 .pTopic = aMQTTTopicName,
+													 .nLinkID = ADI_WIFI_CONNECTION_ID,
+													 .nQos = ADI_WIFI_MQTT_PUBLISER_QOS};
+
```

Comment:
alignment looks strange here, you are too far to the right; the limit on the length of lines is 80 columns and this is a strongly preferred limit.